dvc/repo/plots/template.py

```diff
@@ -27,13 +27,7 @@ class Template:
     INDENT = 4
     SEPARATORS = (",", ": ")
     EXTENSION = ".json"
-    ANCHOR = '"<DVC_METRIC_{}>"'
-    METRIC_DATA_ANCHOR = "<DVC_METRIC_DATA>"
-    X_ANCHOR = "<DVC_METRIC_X>"
-    Y_ANCHOR = "<DVC_METRIC_Y>"
-    TITLE_ANCHOR = "<DVC_METRIC_TITLE>"
-    X_LABEL_ANCHOR = "<DVC_METRIC_X_LABEL>"
-    Y_LABEL_ANCHOR = "<DVC_METRIC_Y_LABEL>"
+    ANCHOR = "<DVC_METRIC_{}>"
```

Comment:
Hi. Does this affect docs? https://dvc.org/doc/command-reference/plots#dvc-template-anchors mentions all of these ATM. Cc @pared